doc/source/user_guide/arrays.rst

```diff
@@ -0,0 +1,50 @@
+
+Handling arrays between PyAnsys Math and Python
+===============================================
+
+
+Sending arrays to PyAnsys Math
+------------------------------
+
+.. code:: ipython3
+
+    import numpy as np
+    import ansys.math.core.math as pymath
+
+    # Start PyAnsys Math as a service.
+    mm = pymath.AnsMath()
+
+    a = np.random.random((2, 3))
+    a_pymath = mm.matrix(a, name="A")
+
+    print(a_pymath)
+
+.. rst-class:: sphx-glr-script-out
```

Comment:
why this?

Reply:
This is to render like this: 
![image](https://user-images.githubusercontent.com/78221213/231387281-35853193-7a1f-4091-a364-2b7b85163e83.png)